src/app/dim-ui/PressTip.m.scss

```diff
@@ -38,34 +32,35 @@ $horizontal-padding: 11px;
     content: '';
     display: block;
     position: absolute;
-    background-color: var(--tooltip-ribbon-color);
-    height: $ribbon-width;
-    top: -$ribbon-width;
+    background-color: var(--theme-tooltip-border-color);
+    height: $theme-tooltip-corner-radius;
+    top: calc(-1 * $theme-tooltip-corner-radius);
```

Comment:
Shouldn't need `calc` here

Reply:
This was a conversion of the existing styles — where it gives a negative top inset to the ribbon — which after removing reveals we're layering 2 ribbons…(?) 

<img width="339" alt="image" src="https://github.com/DestinyItemManager/DIM/assets/1396158/6a55dd2a-be56-43ba-86a5-9f5a226a3c61">

@vivekhnz Can you shed any light? I'm thinking it would be cleaner to remove the `::after` psuedo-element entirely but i'm not sure if this is a fix for a specific instance of a tooltip? I can't seem to recreate the artefacts mentioned in the commit message from #8638 

```  
// The 'ribbon' at the top of the tooltip. We can't use border or an inset box shadow because the change in
// color between the left/right and top edges results in weird slanted artifacts.
```

Reply:
I'm happy for the `::after` pseudo-element to be removed if we can figure out a way to avoid the slanted corner artifacts. These artifacts appear when the top border colour differs from the side border colours. See [this comment](https://github.com/DestinyItemManager/DIM/pull/8638#issuecomment-1216522351) for details.

It looks like the functionality to change the ribbon colour isn't working (or possibly deliberately disabled?) in this branch, which is why it isn't showing up. The relevant mixins are [here](https://github.com/DestinyItemManager/DIM/blob/master/src/app/dim-ui/_tooltip-mixins.scss), and we consume them like [this](https://github.com/DestinyItemManager/DIM/blob/master/src/app/item-popup/PlugTooltip.m.scss#L69).

An easy way to see the difference is to hover over a Fragment in a subclass item popup:
![dim-tooltip-ribbon-colour-bug](https://github.com/DestinyItemManager/DIM/assets/17512262/3842c876-695b-41f9-b27a-f0cd4cc279a5)

Reply:
Ah, thanks! Missing colour is not intentional, will fix that. 

Explains why I couldn't see the artefacts — the `::after` is needed to avoid them. 

I'll reword the css comment to (hopefully) make it a little more explicit what artefacts are being avoided without having to refer back to the previous PR comments.